target/linux/ar71xx/base-files/etc/board.d/01_leds

```diff
@@ -325,6 +325,10 @@ rb-mapl-2nd)
 	ucidef_set_led_netdev "lan" "LAN" "rb:green:eth" "eth0"
 	ucidef_set_led_wlan "wlan" "WLAN" "rb:green:wlan" "phy0tpt"
 	;;
+rb-wap-2nd)
+        ucidef_set_led_timer "user" "USER" "rb:green:user" "1000" "1000"
+	ucidef_set_led_wlan "wlan" "WLAN" "rb:green:wlan" "phy0tpt"
+	;;
```

Comment:
It looks the indention is wrong here.

Reply:
Hm, when I look at file instead of this preview it looks normal.
But when looking at the preview or in a text editor it is indented wrongly.
Will fix it now